### ISSUE TITLE
Source code version inconsistencies ?

### ISSUE THREAD

**Issue by RaiKoHoff** [pypyodbc/pypyodbc#18]
Comparing this project latest release version v1.3.6 (https://github.com/pypyodbc/pypyodbc/releases/tag/v1.3.6)
and PyPi latest version v1.3.6 (https://pypi.org/project/pypyodbc/#files) show up several differences (ignoring whitespaces).

**Comment by RaiKoHoff** [comment]
Maybe related to: https://github.com/pypyodbc/pypyodbc/issues/9 ?

**Comment by rolweber** [comment]
Could you please list the differences you noticed?

**Comment by RaiKoHoff** [comment]
Attached, you will find both source code variants and the patch-file showing the difference.
[_pypyodbc.zip](https://github.com/pypyodbc/pypyodbc/files/9415833/_pypyodbc.zip)

**Comment by rolweber** [comment]
This looks indeed suspicious. And the releases page in GitHub shows release 1.3.6 below 1.3.6rc1, as if it was from an earlier commit:
https://github.com/pypyodbc/pypyodbc/releases

Looks like I messed up when I cut that release, despite the checklist :-(

![image](https://user-images.githubusercontent.com/12234414/186450733-3237ce4c-af6b-4f9e-b7ae-2b09179ffb43.png)

**Comment by rolweber** [comment]
Indeed, I cut the release 1.3.6 from the main branch instead of the 1.3.x branch! &nbsp; _facepalm_

No idea yet when I will find the time to do another, proper release.

**Comment by rolweber** [comment]
Maybe it's not as dramatic as I thought yesterday. Could be that I published the release on PyPI correctly from the 1.3.x branch, but then tagged the wrong commit here in the repo.

@RaiKoHoff Would you have time to run another diff, of the PyPI package against the tip of the 1.3.x branch?
https://github.com/pypyodbc/pypyodbc/tree/1.3.x

**Comment by RaiKoHoff** [comment]
Sorry for the delay.
The current tip of the 1.3.x branch and the current PyPI release contain identical pypyodbc.py files. 👍

**Comment by rolweber** [comment]
Thanks! I'll try to find some time for fixing our tags then.
